src/annotator/sidebar.js

```diff
@@ -117,27 +121,17 @@ export default class Sidebar extends Host {
   }
 
   _setupGestures() {
-    const toggle = $(this.toolbar.sidebarToggleButton);
-
-    if (toggle[0]) {
+    const toggleButton = this.toolbar.sidebarToggleButton;
+    if (toggleButton) {
       // Prevent any default gestures on the handle.
-      toggle.on('touchmove', e => e.preventDefault());
+      toggleButton.addEventListener('touchmove', e => e.preventDefault());
 
-      // Set up Hammer instance and handlers.
-      const manager = new Hammer.Manager(toggle[0])
+      this._hammerManager = new Hammer.Manager(toggleButton)
         // eslint-disable-next-line no-restricted-properties
-        .on('panstart panend panleft panright', this._onPan.bind(this))
-        // eslint-disable-next-line no-restricted-properties
-        .on('swipeleft swiperight', this._onSwipe.bind(this));
-
-      // Set up gesture recognition.
-      const pan = manager.add(
+        .on('panstart panend panleft panright', this._onPan.bind(this));
```

Comment:
`pan*` events are proprietary to `Hammerjs`, yes? I wonder if it's useful to comment that or if it's just kind of obvious...

Reply:
Yes, the `pan*` events are Hammer-specific. I'd hope that's reasonable to intuit from the context.